src/ol/renderer/layerrenderer.js

```diff
@@ -151,7 +151,7 @@ ol.renderer.Layer.prototype.loadImage = function(image) {
     // listener (a noop if the listener was already registered)
     goog.asserts.assert(imageState == ol.ImageState.IDLE ||
         imageState == ol.ImageState.LOADING);
-    goog.events.listenOnce(image, goog.events.EventType.CHANGE,
+    goog.events.listen(image, goog.events.EventType.CHANGE,
```

Comment:
Note that in the cases of tiles (`ol.Tile`) we at least call `dispose` on expired tiles. See https://github.com/openlayers/ol3/blob/master/src/ol/tilecache.jshttps://github.com/openlayers/ol3/blob/master/src/ol/tilecache.js.